alpakaConfig.cmake

```diff
@@ -301,7 +291,15 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
             OPTION(ALPAKA_CUDA_KEEP_FILES "Keep all intermediate files that are generated during internal compilation steps (folder: nvcc_tmp)" OFF)
 
             IF(ALPAKA_CUDA_COMPILER MATCHES "clang")
-                LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "--cuda-gpu-arch=${ALPAKA_CUDA_ARCH}")
+                LIST(
+                    LENGTH
+                    ALPAKA_CUDA_ARCH
+                    _ALPAKA_CUDA_ARCH_LENGTH)
+                IF(${_ALPAKA_CUDA_ARCH_LENGTH} GREATER 1)
+                    MESSAGE(WARNING "Multi-arch compilation with clang as native compiler not supported!")
```

Comment:
You are right. I did not had a look at the documentation and simply expected that it is not supported right now. I stand corrected. It is even easier than for nvcc.